admin/meetings/102022-brainstorm.md

```diff
@@ -0,0 +1,34 @@
+# Brainstorm meeting
+
+**Team number:**
+26
+
+**Team name:**
+Code cowboys
+
+**Type of meeting:**
+Brainstorm
+
+**Attendence:**
+(7/10) Weiwen Dong, Sepehr Heravi, Fucheng Shang, Genevieve Dietz, Ruoqian Huang, Ziyan Zhu, Chengcheng Zhang
```

Comment:
Alberto, Sami, and Jack sent me their ideas before the meeting, so please include their name as well.